projects/lib/src/chessgame.h

```diff
@@ -137,6 +142,7 @@ class LIB_EXPORT ChessGame : public QObject
 		QSemaphore m_pauseSem;
 		QSemaphore m_resumeSem;
 		GameAdjudicator m_adjudicator;
+		const Tournament* m_tournament;
```

Comment:
The `GameWall` only knows individual games. So I reluctantly added `m_Tournament` to `ChessGame`. I originally thought of a `signal`/`slot` solution to notify `GameWall`.